operations/charge.go

```diff
@@ -51,18 +51,19 @@ func (req *ListCharges) Describe() *internal.Description {
 //
 //	fmt.Println("created charge:", charge.ID)
 type CreateCharge struct {
-	Customer                 string                 `json:"customer,omitempty"`
-	Card                     string                 `json:"card,omitempty"`
-	Source                   string                 `json:"source,omitempty"`
-	Amount                   int64                  `json:"amount"`
-	Currency                 string                 `json:"currency"`
-	Offsite                  omise.OffsiteTypes     `json:"offsite,omitempty"`
-	Description              string                 `json:"description,omitempty"`
-	DontCapture              bool                   `json:"-"` // inverse, since `capture` defaults to true
-	ReturnURI                string                 `json:"return_uri,omitempty"`
-	Metadata                 map[string]interface{} `json:"metadata,omitempty"`
-	ExpiresAt                *time.Time             `json:"expires_at,omitempty"`
-	ZeroInterestInstallments *bool                  `json:"zero_interest_installments,omitempty"`
+	Customer                 string                 	`json:"customer,omitempty"`
+	Card                     string                 	`json:"card,omitempty"`
+	Source                   string                 	`json:"source,omitempty"`
+	Amount                   int64                  	`json:"amount"`
+	Currency                 string                 	`json:"currency"`
+	Offsite                  omise.OffsiteTypes     	`json:"offsite,omitempty"`
+	Description              string                 	`json:"description,omitempty"`
+	DontCapture              bool                   	`json:"-"` // inverse, since `capture` defaults to true
+	ReturnURI                string                 	`json:"return_uri,omitempty"`
+	Metadata                 map[string]interface{} 	`json:"metadata,omitempty"`
+	ExpiresAt                *time.Time             	`json:"expires_at,omitempty"`
+	ZeroInterestInstallments *bool                  	`json:"zero_interest_installments,omitempty"`
+	AuthorizationType		 omise.AuthorizationType	`json:"authorization_type,omitempty"`
```

Comment:
Indentation between field, type and tag doesn't look right.

Reply:
try viewing in vscode it shows properly

Reply:
While it's true that certain editors may provide automatic formatting and display improvements, it's important for us to maintain code readability and consistency across different environments and editors.

Reply:
Fixed